lua/dmacro.lua

```diff
@@ -76,32 +76,25 @@ function _M.guess_macro_2(keys)
 end
 
 do
-  --- @type table<integer, { keys: string[]?, macro: dmacro.Range? }>
-  local buf_states = {}
-  vim.api.nvim_create_autocmd('BufDelete', {
-    group = vim.api.nvim_create_augroup('Dmacro', { clear = true }),
-    callback = function(cx)
-      buf_states[cx.buf] = nil
-    end,
-  })
+  ---@type string[]?
+  local _keys = nil
+  ---@type dmacro.Range?
+  local _macro = nil
```

Comment:
Would you mind to avoid the variable name starting `_` ?
```suggestion
  local dmacro_keys = nil
  ---@type dmacro.Range?
  local dmacro_macro = nil
```